src/dmd/dsymbol.d

```diff
@@ -2531,6 +2531,16 @@ Dsymbol handleSymbolRedeclarations(ref Scope sc, Dsymbol s, Dsymbol s2, ScopeDsy
         if (log) printf(" collision\n");
         return null;
     }
+    /*
+    Handle merging declarations with asm("foo") and their definitions
+    */
+    static void mangleWrangle(Declaration oldDecl, Declaration newDecl)
```

Comment:
A better name would be mergeDecls, then you can throw in other bits that need pulling across from old to new.

Reply:
This is part of a function that merges declarations already.

Reply:
All the more reason to call it mergeDecls. ;-)